api/clients/eigenda_client.go

```diff
@@ -137,11 +150,13 @@ func (m EigenDAClient) putBlob(ctx context.Context, rawData []byte, resultChan c
 		return
 	}
 
+	// TODO: should we just use uint8s directly in the config?
```

Comment:
Or change it to `core.QuorumID`?

Reply:
moved the TODO comment to the config and mentioned `core.QuorumID`: [839ffbb](https://github.com/Layr-Labs/eigenda/pull/780/commits/839ffbb7d800e6f90ad6bbc4fe061eca6cd3e759)
Do we have a policy regarding breaking changes like this? Are we fine with just pulling the trigger right now?